recipes/r-ipo/meta.yaml

```diff
@@ -0,0 +1,106 @@
+package:
+  name: r-ipo
+  version: "1.7.5"
+
+source:
+  fn: 3e99def7e2ad455c075edf65c1923a76583c911b.tar.gz
+  url: https://github.com/glibiseller/IPO/archive/3e99def7e2ad455c075edf65c1923a76583c911b.tar.gz
+
+build:
+  skip: True # [osx]
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: parallel
+requirements:
+  build:
+    - r
+    - bioconductor-xcms
+    - r-rsm
+    - bioconductor-camera
+    #- libnetcdf 4.2.1.1
```

Comment:
I might have missed this in the discussion, but I understood that libnetcdf is a dependency right? Then, it should be in here. However, it should be pinned similar to what we do with boost (see e.g. the recipe clever-toolkit and the file `scripts/env_matrix.yml`.